docs/guide/permissions.md

```diff
@@ -2,7 +2,8 @@
 
 ## Permission overview
 
-A User by default does not have access to anything. To get access they need to inherit Roles, typically assigned to the User Group they belong to.
+A User does not have permissions for any part of the system, unless they are explicitly given access.
```

Comment:
Newly created user? Default users that comes with clean installation have some predefined permissions.